.github/workflows/build-fb-image.yaml

```diff
@@ -8,19 +8,28 @@ on:
       - 'v*'
     paths:
       - ".github/workflows/build-fb-image.yaml"
+      - ".github/workflows/scan-docker-image-action.yaml"
       - "cmd/fluent-watcher/fluentbit/**"
       - "cmd/fluent-watcher/hooks/**"
       - "pkg/filenotify/**"
 
 env:
-  FB_IMG: 'kubesphere/fluent-bit:v2.2.2'
-  FB_IMG_DEBUG: 'kubesphere/fluent-bit:v2.2.2-debug'
+  FB_IMG: "kubesphere/fluent-bit:${{ github.ref_name }}"                # kubesphere/fluent-bit:v2.2.2
+  FB_IMG_DEBUG: "kubesphere/fluent-bit:${{ github.ref_name }}-debug"    # kubesphere/fluent-bit:v2.2.2-debug
+
+  FB_IMG_GHCR: "${{ github.repository }}/fluent-bit:${{ github.ref_name }}"              # fluent/fluent-operator/fluent-bit:v2.2.2
+  FB_IMG_DEBUG_GHCR: "${{ github.repository }}/fluent-bit:${{ github.ref_name }}-debug"  # fluent/fluent-operator/fluent-bit:v2.2.2-debug
 
 jobs:
   build:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     name: Build Image for Fluent Bit
+    outputs:
```

Comment:
Outputs should come from steps really, e.g. the metadata action would give you a way to get this. That way if we modify something (or for PRs, non-main branch, etc.) it will carry on working.